client/concordclient/src/concord_client.cpp

```diff
@@ -35,6 +38,20 @@ ConcordClient::ConcordClient(const ConcordClientConfig& config, std::shared_ptr<
     LOG_INFO(logger_, "Waiting for client pool to connect");
     std::this_thread::sleep_for(std::chrono::seconds(2));
   }
+  for (const auto& replica : config_.topology.replicas) {
+    auto addr = replica.host + ":" + std::to_string(replica.event_port);
+    auto grpc_conn =
+        std::make_shared<GrpcConnection>(addr, config_.subscribe_config.id, /* TODO */ 3, /* TODO */ 3, /* TODO */ 5);
+
+    // TODO: Adapt TRC API to support PEM buffers
+    auto trsc_config = std::make_unique<TrsConnectionConfig>(config_.subscribe_config.use_tls,
```

Comment:
It doesn't make sense to initialize a GrpcConnection with a TrsConnectionConfig. Please rename the latter as well.

Reply:
Thanks @robem ,I will change this type aswell.